libevmasm/KnownState.cpp

```diff
@@ -91,6 +91,12 @@ KnownState::StoreOperation KnownState::feedItem(AssemblyItem const& _item, bool
 	{
 		// can be ignored
 	}
+	else if (_item.type() == AssignImmutable)
+	{
+		resetMemory(); // TODO: is this needed?
```

Comment:
I don't think so, as long as you do not use mload to read from the immutable, it should not be possible to interfere with other memory content.